packages/key-management/src/util/cachedGetPassword.ts

```diff
@@ -1,5 +1,6 @@
 import { GetPassword } from '../types';
-import { Milliseconds } from '../../services';
+
+type Milliseconds = number;
```

Comment:
I just defined this alias again for simplicity. Could also lift to `util` package

Reply:
I thought we deleted this util, but apparently not. I think we should delete it as it's a security hazard.

Reply:
067191f8f4f036cd38d64bd1e4520cd8ba3ae3a7